### ISSUE TITLE
linkMapping does not work with relative path (requires absolute path)

### ISSUE THREAD

**Issue by robstoll** [Kotlin/dokka#289]
See #286 for the discussion

**Comment by semoro** [comment]
@robstoll Nice! Ping if you'll have questions

**Comment by pardom** [comment]
In the meantime, you can work around this by prepending `linkMapping.dir` with `${projectDir}`.

**Comment by ethauvin** [comment]
Don't think this was fixed in 0.9.18 on Windows. Seems to work fine under Linux.

The workaround of prepending `${projectDir)` doesn't work on Windows either, as it then complains about not being a valid Unix path.

The only way I got it to work is by doing something like:

```
dir = file("${projectDir}/src/main/kotlin").toURI().toString().replace("file:", "")
```
